src/test/java/wolox/training/controllers/BookTest.java

```diff
@@ -0,0 +1,173 @@
+package wolox.training.controllers;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.ArgumentMatchers.any;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import static org.mockito.BDDMockito.given;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(BookController.class)
+class BookControllerIntegrationTest {
```

Comment:
The class name is `BookTest` but in code the name is `BookControllerIntegrationTest`. Fix this in all places